logrotate/CMakeLists.txt

```diff
@@ -21,12 +21,25 @@ SET(ACTIVE_CPP0xx_DIR "Release")
 INCLUDE (${LogRotate_SOURCE_DIR}/Options.cmake)
 INCLUDE_DIRECTORIES(${G3_HEADER_PATH})
 
+IF (MSVC)
+    SET(CMAKE_DEBUG_POSTFIX "_d")
```

Comment:
Nice. I like this.  I will try it out and if it works great this is candidate for being for all platforms. I appreciate you made it for windows only in this PR